test/fixtures/fixtureDataCorrect.json

```diff
@@ -0,0 +1 @@
+{"state":"Ukraine"}
```

Comment:
No need to name your file as `fixtureBlaBla` if it us under `fixtures` folder.
The only good naming convention with words repeat is `./fixtures/filename.fixture.json` but think it is redundant as well.

Reply:
done